src/components/widgets/WidgetTeam/WidgetTeam.vue

```diff
@@ -1,88 +1,16 @@
 <template>
   <widget v-if="activeItem.team" :title="$t('general.team')">
     <empty-state
-      v-if="!activeItem.team.length"
+      v-if="!teamMembers"
       :icon="'user-ninja'"
       :heading="$t('empty.team.heading')"
       :body="$t('empty.team.body')"
     />
 
-    <template v-if="director">
+    <template v-for="role in sortByDisplayOrder(Object.keys(this.teamMembers))">
```

Comment:
`this` brukes ikke i templates. Her trenger vi kanskje heller ikke lenger en `template`, men sette `v-for` direkte på `role-members`-komponenten? Mangler også en [`key`](https://v2.vuejs.org/v2/api/#key).